### ISSUE TITLE
Rename standalone to "stripped-api" and api to "with-api" or something similarly less confusing.

### ISSUE THREAD

**Issue by wagyourtail** [cabaletta/baritone#3090]
## Describe your suggestion
It can be confusing as to what "standalone" entails with the different levels of obfuscation in releases for example one might conclude that the "api" version also requires one of the other two since one might think it *only* includes the api.

## Context
#3089 probably
#2979 maybe, complete guess on this one
#2838 technically
#2550 see my comment here
#1119 most likely
more in baritone discord server and in certain deleted servers

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by scorbett123** [comment]
Is the standalone version really necessary, I can't investigate into speed changes at the moment, but are they that much, or should the api just be distributed as default?

**Comment by ZacSharp** [comment]
I'm surprised the "Expected ab/aa/whatever" bug is not a problem with the standalone/api versions, we just should never obfuscate things that are exposed to the user (I have a oneliner fix in mind, give me a bit to test it). So #3089 and #2838 are fixable without changing what artifacts we release.

For the naming I don't think the api jar is the problem, but rather people mistakenly installing standalone (at least from what I see, maybe it's the other way round on discord), so I'd keep api and unoptimized and search for a way to make it more clear that *standalone can not be used as a dependency* or maybe stop releasing it, in case it has only little gain. Most people want some client with baritone integration anyways and except for ForgeHax that means they can't use standalone.

**Comment by wagyourtail** [comment]
> For the naming I don't think the api jar is the problem, but rather people mistakenly installing standalone (at least from what I see, maybe it's the other way round on discord), so I'd keep api and unoptimized and search for a way to make it more clear that *standalone can not be used as a dependency* or maybe stop releasing it, in case it has only little gain. Most people want some client with baritone integration anyways and except for ForgeHax that means they can't use standalone. 

Yes thats exactly the issue I'm proposing we fix by changing the names

**Comment by scorbett123** [comment]
I would support the stopping of releasing standalone at all, it doesn't really have many uses, and I don't think there is many drawbacks to just using the api.

**Comment by ZacSharp** [comment]
Maybe I should have written it more explicitly: I'm not against renaming, I just want to rename *only* standalone.

But since I can't think of a clear name that would prevent that misconception my preferred option currently is not releasing it anymore, unless someone comes  up with a good way to make it clear what the jar is for.
